CourseManager/Courses/Course 1 Build a SwiftUI app for iOS 15/Views/Detail/CourseView.swift

```diff
@@ -66,6 +68,7 @@ struct CourseView: View {
                     .frame(maxWidth: 500)
                     .matchedGeometryEffect(id: "image\(course.id)", in: namespace)
                     .offset(y: scrollY > 0 ? scrollY * -0.8 : 0)
+                    .accessibilityLabel("Cover Image")
```

Comment:
why did you add this?

Reply:
At the end of the course, the instructor touched on accessibility and it probably should read something more descriptive than "Cover Image".  It's just an example of what can be done